lib/src/git_backend.rs

```diff
@@ -703,6 +759,29 @@ impl Backend for GitBackend {
     }
 }
 
+/// Write a tree conflict as a special tree with `.jjconflict-base-N` and
+/// `.jjconflict-base-N` subtrees. This ensure that the parts are not GC'd.
+fn write_tree_conflict(
```

Comment:
Nice!

Reply:
This will, unfortunately, be very visible to people in a colocated repo. VS Code will try to show all the "changed files" when editing a conflict resolution.

I'm not sure if there's anything to do about this, but we should mark it in the changelog when we switch to using tree-level conflicts by default.